integration/simulation/network/geth_network.go

```diff
@@ -0,0 +1,77 @@
+package network
+
+import (
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
+
+	"github.com/obscuronet/obscuro-playground/go/buildhelper"
+	"github.com/obscuronet/obscuro-playground/go/ethclient"
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/host"
+	"github.com/obscuronet/obscuro-playground/integration/simulation/p2p"
+	"github.com/obscuronet/obscuro-playground/integration/simulation/params"
+	"github.com/obscuronet/obscuro-playground/integration/simulation/stats"
+)
+
+type networkInMemGeth struct {
+	obscuroNodes     []*host.Node
+	obscuroAddresses []string
+}
+
+func NewNetworkInMemoryGeth() Network {
+	return &networkInMemGeth{}
+}
+
+// Create inits and starts the nodes, wires them up, and populates the network objects
+func (n *networkInMemGeth) Create(params params.SimParams, stats *stats.Stats) ([]ethclient.Client, []*host.Node, []string) {
```

Comment:
@otherview Can you add a TODO to decompose this stuff, so that we can pick and choose what type of Obscuro hosts, Obscuro nodes and Eth nodes are running against each other? e.g. allowing us to select out-of-process Geth nodes with out-of-process Obscuro nodes.

Reply:
That's a good point. Added 👍